tests/test_wait.py

```diff
@@ -41,3 +41,9 @@ def condition():
         return wait().until(lambda: True)
 
     assert wait().until(condition)
+
+
+def test_wait_until_with_lambda_captures():
```

Comment:
This test pass without your fix as well, which makes it redundant

Reply:
fixed